prediction_market_agent/agents/blockchain_coding_agent/run_agent04_read.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
```

Comment:
Can you create a scripts directory like `prediction_market_agent/agents/blockchain_coding_agent/scripts` and move these run files there? Just to keep clean structure about what is a script and what not